canvas/circle.go

```diff
@@ -41,6 +41,9 @@ func (c *Circle) MinSize() fyne.Size {
 
 // Move the circle object to a new position, relative to its parent / canvas
 func (c *Circle) Move(pos fyne.Position) {
+	if c.Position1.X == pos.X && c.Position1.Y == pos.Y {
```

Comment:
I think you can do `c.Position == pos` instead. The same applies to all places below.